src/docs/product/metrics/index.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: "Metrics"
+sidebar_order: 55
+description: "Sentry's Developer Metrics help you track and visualize the data points you care about over time, making it easier to monitor your application health and identify issues."
+---
+
+<Include name="feature-stage-alpha-metrics.mdx" />
+
+Sentry metrics are numerical values that allow you to pinpoint and solve issues impacting user experience and app performance by measuring the data points that are important to you and your application. You can track things like latency, processing time, event size, user signups, and conversion rates, and see how they correlate back to tracing data. This gives you a comprehensive understanding of the health of your systems and helps solve issues faster.
+
+![Metrics UI](Index.png)
+
+## Metric Types
+
+Metrics at Sentry come in different flavors, in order to help you track your data in the most efficient and cost-effective way. The metric types we currently support are:
+
+- **Counters:** Tracks a value that can only be incremented (for example, button clicks)
+- **Distributions:** Tracks a list of values which can be aggregated over time like `max`, `min`, `avg` (for example, page load times)
+- **Gauges:** Tracks a value that can go up or down (for example, available disk space, memory used)
+- **Sets:** Tracks a set of values on which can be aggregations over time such as `count_unique` (for example, number of unique users)
+
+Each metric also needs to have a unit associated with it, so that you know what you're dealing with. Examples of units are seconds, milliseconds, bytes, or even potatoes if you like.
+
+## Augmenting Metrics with Tags
+
+Metrics are powerful on their own, but you can enrich them further by adding dimensions in the form of Tags. These are key/value string pairs (for example, `cuisine:Italian`) that are associated with metrics to provide contextual information, and are often used to filter and group them during analysis.
+
+Sentry adds certain common tags by default such as, `project`, `environment`, and `release`, but you can also create your own custom tags to track attributes that help organize your data for your specific use case. Some common examples of useful tags are browser name, region, language, or customer.
```

Comment:
`project` is not a custom tag. It adds by default `transaction`, `environment`, `release`.